### ISSUE TITLE
"ALTER OPERATOR FAMILY" returns an error

### ISSUE THREAD

**Issue by dhanashreek89** [greenplum-db/gpdb#1538]
When we alter operation family, gpdb returns an error.
Repro:

```
CREATE OPERATOR FAMILY alt_opf17 USING btree;
ALTER OPERATOR FAMILY alt_opf17 USING btree ADD
OPERATOR 1 < (int4, int2) ,
  OPERATOR 2 <= (int4, int2) ,
  OPERATOR 3 = (int4, int2) ,
  OPERATOR 4 >= (int4, int2) ,
  OPERATOR 5 > (int4, int2) ,
  FUNCTION 1 btint42cmp(int4, int2);

ERROR:  no pre-assigned OID for pg_amop tuple "" (oid_dispatch.c:539)  (seg1 127.0.0.1:25433 pid=10291)
```

The above repro passes on Postgres 9.6.1

**Comment by danielgustafsson** [comment]
Thanks for the report, I'll have a look.

**Comment by hlinnaka** [comment]
Fixed by PR #1546, closing issue.
